references-to-score.rb

```diff
@@ -0,0 +1,21 @@
+SHA = "393d678718de02a15ba8c4894529c9494151211f"
```

Comment:
I see this is in the root directory, what is this file for?

Reply:
Hmm…that's a script for a previous query I was running.  It does not belong to this PR.  I'll remove.